Vendors/tropter/cmake/CMakeLists.txt

```diff
@@ -5,6 +5,16 @@ if(TROPTER_COPY_DEPENDENCIES AND APPLE)
     file(GLOB gfortran "${gcc_libdir}/libgfortran*.dylib")
     file(GLOB quadmath "${gcc_libdir}/libquadmath*.dylib")
 
+    # Get the actual name of the library libgcc_s.1.dylib (can be libgcc_s.1.1.dylib),
+    # or another if it changes in the future.
+    execute_process(COMMAND bash "-c"
+        "basename `otool -L ${IPOPT_LIBDIR}/libcoinmumps.dylib  | grep 'libgcc' | awk '{print $1}'`"
+        OUTPUT_VARIABLE libgcc_name
+        OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+    
+    message(STATUS "NAME: ${libgcc_name}")
+
```

Comment:
This message is helpful for debugging but could be removed since this rarely changes